client/signup/steps/plans-theme-preselected/index.tsx

```diff
@@ -40,7 +45,31 @@ function getHidePlanPropsBasedOnSignupDependencies(
 function PlansThemePreselectedStep( props: object & { signupDependencies: SignupDependencies } ) {
 	const hidePlanProps = getHidePlanPropsBasedOnSignupDependencies( props.signupDependencies );
 
-	return <PlansStep { ...props } { ...hidePlanProps } />;
+	const freePlanButton = (
+		<Button
+			onClick={ () => buildUpgradeFunction( { ...props, recordTracksEvent }, null ) }
+			borderless
+		/>
+	);
+
+	let subHeaderText = translate(
+		"Pick one that's right for you and unlock features that help you grow. Or {{link}}start with another theme{{/link}}.",
+		{ components: { link: freePlanButton } }
+	);
+
+	if ( ! isDesktop() ) {
+		subHeaderText = translate( 'Choose a plan or {{link}}start with another theme{{/link}}.', {
+			components: { link: freePlanButton },
+		} );
+	}
+
+	return (
+		<PlansStep
+			{ ...props }
+			{ ...hidePlanProps }
+			fallbackSubHeaderText={ preventWidows( subHeaderText ) }
```

Comment:
I still see the dot in a separate line:
<img width="524" alt="Screenshot 2023-05-11 at 11 03 00" src="https://github.com/Automattic/wp-calypso/assets/1233880/f16833ec-6848-4bbf-b673-07229bec901e">